js/foam/browser/u2/BrowserView.js

```diff
@@ -19,431 +19,359 @@ CLASS({
   name: 'BrowserView',
   extends: 'foam.u2.View',
   requires: [
-    'foam.browser.BrowserConfigU2',
-    'foam.browser.u2.StackView',
-    'foam.input.touch.GestureManager',
-    'foam.input.touch.TouchManager',
-    'foam.u2.ElementParser',
+    'foam.u2.DAOController',
+    'foam.u2.DAOCreateController',
+    'foam.u2.PropertyView',
+    'foam.u2.SearchBorder',
     'foam.u2.md.ActionButton',
-    'foam.u2.md.Checkbox',
     'foam.u2.md.Input',
     'foam.u2.md.Select',
-    'foam.u2.md.SharedStyles',
-    'foam.u2.md.TextArea'
   ],
-
   imports: [
-    'gestureManager',
-    'touchManager',
+    'document',
+    'headerColor',
+    'menuFactory',
+    'selection$',
+    'stack',
   ],
   exports: [
-    'gestureManager',
-    'stack',
-    'touchManager',
+    'headerColor',
+    'selection',
   ],
 
-  models: [
+  properties: [
+    {
+      name: 'data',
+      documentation: 'This is the DAO to view.',
+    },
+    {
+      name: 'title',
+      defaultValueFn: function() { return this.data.model.name + ' Browser'; }
+    },
+    {
+      type: 'Boolean',
+      name: 'showAdd',
+      documentation: 'Set this to false to hide the create button.',
+      defaultValue: true
+    },
+    {
+      name: 'headerColor',
+      factory: function() {
```

Comment:
Ah, yeah, it does. That was an editing quirk; there used to be more logic in there.